packages/@lightningjs/ui-components/src/docs/Base.mdx

```diff
@@ -80,7 +80,7 @@ Any component which extends the Base component and uses the `withThemeStyles` mi
 This method accepts a target component, patch object, and optional smooth object. If the component is visible, it will smooth in the smooth object, or fall back to
 the patch object, if not it will apply the patch.
 
-#### isFullyOnScreen(): bool
+#### isFullyOnScreen({ offsetX: number, offsetY: number }): bool
```

Comment:
getting a build error on this file
need to add \ in front of both curly braces

Reply:
fixed, thanks!